book/installation.rst

```diff
@@ -228,6 +228,12 @@ If there are any issues, correct them now before moving on.
     configuration (e.g. commonly httpd.conf or apache2.conf for Apache) and setting
     its user to be the same as your CLI user (e.g. for Apache, update the ``User``
     and ``Group`` values).
+    
+    .. caution::
+    
+      If used in a production environment, be sure this user only has limited privileges
+      (no access to private data or servers, launch of unsafe binaries, etc.)
+      as a compromised server would give to the hacker those privileges.
```

Comment:
It was working, but you're right, and it's better for readability.
I also rebased as my other PR on the same file has just been merged.